pyradiosky/tests/test_skymodel.py

```diff
@@ -2098,13 +2098,23 @@ def test_order_healpix_to_sky(healpix_data, hpx_order):
             assert sky.hpx_order == hpx_order
 
 
-@pytest.mark.filterwarnings("ignore:recarray flux columns will no longer be labeled")
 def test_healpix_recarray_loop(healpix_disk_new):
 
     skyobj = healpix_disk_new
-    skyarr = skyobj.to_recarray()
+    with uvtest.check_warnings(
+        DeprecationWarning,
+        match=[
+            "The to_recarray method is deprecated and will be removed in 0.3.0.",
```

Comment:
To confirm -- Is the plan to deprecate `array_to_skymodel` in 0.2.0 and then `to_recarray` in 0.3.0?

Reply:
yes. There's also a `from_recarray` which was the replacement for `array_to_skymodel`, which will also be deprecated in 0.3.0.